user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,886 @@
+
+--
+
+Linux::
++
+--
+[source,bash,subs="attributes+"]
+----
+wget {release_url} <1>
+tar xvfj {release_filename} <2>
+cd {release_basename} <3>
+----
+
+<.> Download the boost distribution `{release_filename}`
+<.> Extract the files into `{release_basename}`
+<.> Change the current working directory to `{release_basename}`
+--
+
+macOS::
++
+--
+[source,bash,subs="attributes+"]
+----
+curl {release_url} <1>
+tar xvfj {release_filename} <2>
+cd {release_basename} <3>
+----
+
+<.> Download the boost distribution `{release_filename}`
+<.> Extract the files into `{release_basename}`
+<.> Change the current working directory to `{release_basename}`
+--
+
+Git::
++
+--
+You can clone and initialize the complete Boost super-project directly from GitHub with:
+
+[source,bash,subs="attributes+"]
+----
+git clone https://github.com/boostorg/boost.git -b boost-{latest_version} {release_basename} --depth 1 <1>
+cd {release_basename}
+git submodule update --depth 1 --init --recursive <2>
+----
+
+<.> Clone the Boost super-project
+<.> Initialize all boost modules
+--
+====
+
+==== Installing Individual Boost Modules
+
+The process above downloads all boost libraries, which is the usual process in a local installation.
+In projects where only a subset of the Boost libraries is required, the following procedure can be used:
+
+[source,bash,subs="attributes+"]
+.Downloading boost:unordered[] and internal dependencies
+----
+git clone https://github.com/boostorg/boost.git -b boost-{latest_version} {release_basename} --depth 1 <1>
+git submodule update --depth 1 -q --init tools/boostdep <2>
+git submodule update --depth 1 -q --init libs/unordered <3>
+python tools/boostdep/depinst/depinst.py -X test -g "--depth 1" unordered <4>
+----
+
+<1> Download the Boost super-project, which contains references to all Boost libraries
+<2> Initialize boost:boostdep[] used to determine internal boost dependencies
+<3> Initialize boost:unordered[].
+Repeat this step for any other modules you need.
+<4> Initialize any dependencies of boost:unordered[].
+Repeat this step for any other modules you need.
+
+This procedure requires `git` and `python`.
+The following steps in this document apply to both methods of installing Boost.
+The only difference is what libraries will be available.
+
+=== Bootstrap
+
+If you followed the steps above, your current working directory should be `{release_basename}`.
+Otherwise, open up a command line terminal, and navigate to the boost folder you extracted.
+
+[tabs,sync-group-id=build]
+====
+Boost.Build::
++
+--
+The Boost libraries are built using a custom build app called boost:build[].
+This app itself is built by running the bootstrap script.
+
+This is the usual procedure to configure, build and install Boost with boost:build[]:
+
+Decide where you are going to install the libraries to, most locations are OK, something like *Documents/Boost-libraries* is fine.
```

Comment:
shouldn't the commands use this? Or remove it alltogether, I think our users are old enough to know this :)

Reply:
Yes. :laughing: I also thought about that. But I was just merging the content we had from 5 guides we had and concluded I would never finish this if I kept fixing the content. There are many weird sentences like that.